### PR TITLE
Move the custom ACES Python utils out of BSK

### PR DESCRIPTION
**Pull request by galensavidge**
I also made the aces Python code a package so that we don't have to add directories to the path at the top of scripts. Sims and unit tests all still run. I might have missed a reference or two somewhere but it'll be easy to fix if we find issues.

### CODE REVIEW

**Comment by alboehm** [comment]
leaving this one for Jeremiah bc he's more familiar with the aces structure

File: aces/ExternalModules/attitudeEKF/_UnitTest/test_attitudeEKF.py [hunk 1]
```diff
@@ -47,18 +50,17 @@ def setupStarTrackerMsg(starTrackerInMsgData, starTrackerInMsg, timeTag = validT
...
     gyroInMsg.write(gyroInMsgData)
-    
+
 def setupValidEkfConfig(attitudeEkfConfig, unitTestSim):
     unitTestSim.InitializeSimulation()
-    ARCSEC2DEG = 1 / 3600.  # conversion from arcseconds to degrees
-    aroundBoresightStd = 30 * ARCSEC2DEG * macros.D2R
-    crossBoresightStd = 10 * ARCSEC2DEG * macros.D2R
+    aroundBoresightStd = 30 * acesmacros.ARCSEC2DEG * macros.D2R
+    crossBoresightStd = 10 * acesmacros.ARCSEC2DEG * macros.D2R
```
Comment (JeremiahMunson) [inline aces/ExternalModules/attitudeEKF/_UnitTest/test_attitudeEKF.py:63]:
```suggestion
    aroundBoresightStd = 30 * acesmacros.ARCSEC2RAD
    crossBoresightStd = 10 * acesmacros.ARCSEC2RAD
```

File: aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py [hunk 2]
```diff
@@ -6,18 +8,21 @@
 #
 
 import pytest
+import matplotlib.pyplot as plt
+import numpy.matlib
```
Comment (JeremiahMunson) [inline aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py:12]:
```suggestion
```
Not used in this file

File: aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py [hunk 3]
```diff
@@ -103,69 +108,68 @@ def starTrackerSensorTestFunction(showPlots, accuracy):
     # Setup the Star Tacker.
     starTracker = starTrackerSensor.StarTrackerSensor()
     starTracker.ModelTag = "starTrackerSensor"
-    ARCSEC2DEG = 1 / 3600.  # conversion from arcseconds to degrees
-    aroundBoresightStd = 30 * ARCSEC2DEG * macros.D2R
-    crossBoresightStd = 10 * ARCSEC2DEG * macros.D2R
+    aroundBoresightStd = 30 * acesmacros.ARCSEC2DEG * macros.D2R
+    crossBoresightStd = 10 * acesmacros.ARCSEC2DEG * macros.D2R
```
Comment (JeremiahMunson) [inline aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py:112]:
```suggestion
    aroundBoresightStd = 30 * acesmacros.ARCSEC2RAD
    crossBoresightStd = 10 * acesmacros.ARCSEC2RAD
```

File: aces/ExternalModules/attitudeEKF/_UnitTest/test_attitudeEKF.py (hunk 1 above)
Comment (galensavidge) [reply aces/ExternalModules/attitudeEKF/_UnitTest/test_attitudeEKF.py]: Good catch! Fixed

File: aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py (hunk 2 above)
Comment (galensavidge) [reply aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py]: Fixed, not sure how this got in there

File: aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py (hunk 3 above)
Comment (galensavidge) [reply aces/ExternalModules/starTrackerSensor/_UnitTest/verification_starTrackerSensor.py]: Fixed

**Review by JeremiahMunson — APPROVED** [review]

**Pull request merged**